website/docs/r/branch_protection.html.markdown

```diff
@@ -49,7 +48,11 @@ resource "github_branch_protection" "example" {
 
 }
 
-resource "github_user" "example" {
+resource "github_repository" "example" {
```

Comment:
This resource is needed to make the example code work; I've included it as the examples generally include a _full_ rundown of resources.